docs/metrics/nimbus/metrics.md

```diff
@@ -0,0 +1,33 @@
+<!-- AUTOGENERATED BY glean_parser.  DO NOT EDIT. -->
```

Comment:
I suspect the changes to all the other metrics.md files are due to glean being updated - but I can't explain why this new file!

Reply:
It's definitely a little weird how it would pop up as a brand new file. Especially since it generates even on a clean branch as well